website/_containers/dataviz/2013-01-04-part-1.md

```diff
@@ -42,7 +42,7 @@ In order to read a CSV/Excel file, we have to import the `csv` module from Pytho
 import csv
 ```
 
-`MY_FILE` is defining a global - notice how it‘s all caps, a convention for variables we won't be changing. Included in this repo is a sample file to which this variable is assigned.
+`MY_FILE` is defining a constant - notice how it‘s all caps, a convention for variables we won't be changing the value of. Included in this repo is a sample file to which this variable is assigned.
```

Comment:
They are called constants though in PEP-0008: https://www.python.org/dev/peps/pep-0008/#constants